romancal/refpix/data.py

```diff
@@ -224,6 +224,13 @@ def remove_offset(self) -> StandardView:
 
         # Apply the offset to the data (in place)
         data -= offset
+        m = (data == 0) & (offset != 0)
+        data[m] = 1e-9  # hack!  But remove_trends uses data = 0 as a signal
+        # for missing data, so data whose offset subtracts exactly is treated
+        # as missing rather than as truly zero, which is bad.  This perturbs
+        # that slightly.
+        # it would be better to have the frame_not_zero checks look at
+        # the offsets explicitly.
```

Comment:
```suggestion
        # HACK!
        # `remove_trends` uses data = 0 as a signal for missing data, so
        #    data whose offset subtracts exactly is treated as missing rather
        #    than as truly zero, which is bad.  This perturbs that slightly.
        # It would be better to have the frame_not_zero checks look at
        #    the offsets explicitly.
        data[m] = 1e-9
```
This cleans up the comment a bit. It makes it make more sense to me.